nicsdru_nidirect_theme.theme

```diff
@@ -328,16 +328,36 @@ function nicsdru_nidirect_theme_preprocess_page_title(&$variables) {
   // By default page titles are visible.
   $variables['title_visible'] = TRUE;
 
-  // Preprocess node titles.
-  if ($node = \Drupal::routeMatch()->getParameter('node')) {
-
-    // Landing pages have a boolean field to enable title visibility.
-    if ($node->getType() == 'landing_page') {
-      // Is the title visible?
-      if ($node->field_enable_title->get(0)->value === '0') {
-        $variables['title_visible'] = FALSE;
+  // Preprocess titles on certain routes.
+  $route = \Drupal::routeMatch();
+
+  switch ($route->getRouteName()) {
+
+    // Preprocess node titles.
+    case "entity.node.canonical":
+      $node = $route->getParameter('node');
+      // Landing pages have a boolean field to enable title visibility.
+      if ($node->getType() === 'landing_page') {
+        // Is the title visible?
+        if ($node->field_enable_title->get(0)->value === '0') {
+          $variables['title_visible'] = FALSE;
+        }
       }
-    }
+      break;
+
+    // Preprocess news listing page title.
+    case "nidirect_news.news_listing":
+      // Add a subtitle to indicate the current page selected from the pager.
+      $page = \Drupal::request()->get('page');
+      if (isset($page) && $page > 0) {
+        $page = ++$page;
+        $variables['title_subtitle'] = t('page :page', [':page' => $page]);
+      }
+      break;
+
+    default:
+      break;
```

Comment:
I think that this is wasted code really, you don't need a 'default' case that only contains a 'break' - you can just delete both lines

Reply:
Done.